src/Core/Feature.js

```diff
@@ -77,49 +79,44 @@ export class FeatureGeometry {
     /**
      * Push new coordinates in vertices buffer.
      * @param {Coordinates} coord The coordinates to push.
+     * @param {Feature} feature - the feature containing the geometry
      */
-    pushCoordinates(coord) {
-        if (coord.crs !== this._feature.crs) {
-            coord.as(this._feature.crs, coord);
+    pushCoordinates(coord, feature) {
+        if (coord.crs !== feature.crs) {
+            coord.as(feature.crs, coord);
         }
-        if (this._feature.normals) {
-            coord.geodesicNormal.toArray(this._feature.normals, this._feature._pos);
+        if (feature.normals) {
+            coord.geodesicNormal.toArray(feature.normals, feature._pos);
         }
 
-        this._feature._pushValues(this._feature, coord.x, coord.y, coord.z);
+        feature._pushValues(feature, coord.x, coord.y, coord.z);
```

Comment:
it's possible to remove `feature` from `_pushValues`  params?

Reply:
Done